invenio_administration/assets/semantic-ui/js/invenio_administration/api/config.js

```diff
@@ -0,0 +1,43 @@
+// This file is part of Invenio-requests
+// Copyright (C) 2022 CERN.
+//
+// Invenio-requests is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import axios from "axios";
+
+const apiConfig = {
+  withCredentials: true,
+  xsrfCookieName: "csrftoken",
+  xsrfHeaderName: "X-CSRFToken",
+  headers: {
+    Accept: "application/json",
```

Comment:
Tiny minor ocd comment (😄) : use double quotes as below just for consistency

```suggestion
    "Accept": "application/json",
```